content/main.js

```diff
@@ -369,7 +454,26 @@ var tblatex = {
     editor.endTransaction();
   };
 
+  tblatex.on_middleclick = function(event) {
```

Comment:
also for a future PR: middle click is usually used to paste text from the X clipboard on Unix so you may run into a conflict here

Reply:
Yes, but this is just for the `Latex It!` button, the middle click in the text area is still working (at least for me).

Reply:
ah of course -- 👍

Reply:
I don't know what the status is of the documentation or something but this is probably something that should be at least written up on the wiki, if you haven't done so already